live-examples/css-examples/cursor/cursor.css

```diff
@@ -0,0 +1,15 @@
+.example-choice-list {
+  height: 500px;
+  width: 250px;
+}
+
+.cursor {
+  display: flex;
+  background-color:#1766aa;
+  color: white;
+  height: 100px;
+  width: 200px;
+  justify-content: center;
+  align-items: center;
+  font-size: 14pt;
+}
```

Comment:
We use 4 spaces to indent (sorry not to have a style guide linked from the contribution docs).